src/libraries/System.Memory/tests/Span/EnumerateRunePositions.cs

```diff
@@ -0,0 +1,206 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        private static void RunePosition_TestProps(Rune rune, int startIndex, int length, bool wasReplaced, RunePosition runePosition)
+        {
+            Assert.Equal(rune, runePosition.Rune);
+            Assert.Equal(startIndex, runePosition.StartIndex);
```

Comment:
When adding the RunePosition public constructor, this constructor can be used here and the calls Equals?